core/src/main/scala/org/apache/spark/sql/streaming/DirectKafkaStreamSource.scala

```diff
@@ -16,63 +16,135 @@
  */
 package org.apache.spark.sql.streaming
 
-import scala.reflect.ClassTag
-
-import kafka.serializer.Decoder
-
+import org.apache.kafka.common.TopicPartition
 import org.apache.spark.Logging
 import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.encoders.RowEncoder
 import org.apache.spark.sql.sources.{BaseRelation, DataSourceRegister}
 import org.apache.spark.sql.types.StructType
 import org.apache.spark.streaming.dstream.DStream
-import org.apache.spark.streaming.kafka.KafkaUtils
-import org.apache.spark.util.Utils
+import org.json4s.NoTypeHints
+import org.json4s.jackson.Serialization
+import scala.collection.mutable.HashMap
+import scala.util.control.NonFatal
+import org.apache.spark.streaming.kafka010.{ConsumerStrategies, KafkaUtils, LocationStrategies}
 
 class DirectKafkaStreamSource extends StreamPlanProvider with DataSourceRegister {
 
-  override def shortName(): String = SnappyContext.DIRECT_KAFKA_STREAM_SOURCE
+  override def shortName(): String = SnappyContext.KAFKA_STREAM_SOURCE
 
   override def createRelation(sqlContext: SQLContext,
-      options: Map[String, String],
-      schema: StructType): BaseRelation = {
+                              options: Map[String, String],
+                              schema: StructType): BaseRelation = {
     new DirectKafkaStreamRelation(sqlContext, options, schema)
```

Comment:
Our default code-style formatting should not lead to this. Please revert these indentation changes here and below and ensure that codeStyleSettings.xml is setup correctly in IDEA as documented.